CONTRIBUTING.md

```diff
@@ -200,6 +207,8 @@ For datasets in particular, these should be loaded using a `loader` function fro
 
 For methods and metrics, they should be decorated with the appropriate function in `openproblems.tools.decorators` to include metadata required for the evaluation and presentation of results.
 
+For methods and metrics requiring image-specific packages, see the "Adding Package Dependencies" section for further explanation.
```

Comment:
This feels a little redundant given the section is question is above this one, no?

Reply:
Bump. What do you think?

Reply:
Bumping this @wes-lewis